src/locales/de/UI/REPORT.json

```diff
@@ -0,0 +1,30 @@
+{
+    "BY": "Erstellt von",
+    "COMMISION_BY": "Beauftragt von",
+    "HD_ABOUT": "Über die Prüfung",
+    "HD_SUMMARY": "Zusammenfassung",
+    "HD_SCOPE": "Umfang der Prüfung",
+    "LABEL_SITE_NAME": "Name der Website",
+    "LABEL_SITE_SCOPE": "Umfang der Website",
+    "LABEL_CONFORMANCE_TGT": "Angestrebte Stufe",
+    "LABEL_EXTRA_REQUIREMENTS": "Zusätzliche Anforderungen an die Prüfung",
+    "LABEL_SUPPORT_BASE": "Referenz für die Unterstützung der Barrierefreiheit",
+    "LABEL_RELIEDUP_TECH": "Verwendete Webtechniken",
+    "LABEL_NOT_PROVIDED": "Nicht angegeben",
+    "HD_SCORE": "Übersicht der Prüfergebnisse",
+    "RESULTS_OF": "Ergebnisse von",
+    "PRINCIPLE": "Prinzip",
+    "TOTAL_SCORE": "Total",
+    "HD_CRITERIA_REPORT": "Ausführliche Prüfergebnisse",
+    "LABEL_DESCR": "Ergebnisse",
+    "HD_SAMPLE": "Geprüfte ausgewählte Seiten",
```

Comment:
Nitpick: I'm not sure what is more correct, but I at least would write this with a "comma", like "Geprüfte, ausgewählte Ergebnisse"